src/factories.py

```diff
@@ -12,6 +14,11 @@
 from tasks.models import Task
 
 
+def _get_random_date():
```

Comment:
nice

Reply:
This is the cause for the failing test. I will have to look into it.

Reply:
This is built in to factoryboy via faker, so you don't need this helper function.
example:
https://github.com/codalab/chahub/blob/e66203d36ae43802b2f304a0564c941bc9b1e174/src/factories.py#L148

Reply:
(to fix tests, you'll probably just need to add `created_when=timezone.now()` to the test (maybe a helper function) that is creating the submissions)

Reply:
Great call, Jim. Much cleaner! ✅